src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/refreshanalyzer/RestRefreshSearchAnalyzerActionIT.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexmanagement.refreshanalyzer
+
+import com.amazon.opendistroforelasticsearch.indexmanagement.IndexManagementPlugin.Companion.ANALYZER_BASE_URI
+import com.amazon.opendistroforelasticsearch.indexmanagement.IndexManagementRestTestCase
+import com.amazon.opendistroforelasticsearch.indexmanagement.indexstatemanagement.makeRequest
+import org.elasticsearch.client.ResponseException
+import org.elasticsearch.common.settings.Settings
+import org.elasticsearch.rest.RestRequest.Method.POST
+import org.elasticsearch.rest.RestStatus
+
+class RestRefreshSearchAnalyzerActionIT : IndexManagementRestTestCase() {
+
+    fun `test missing indices`() {
+        try {
+            client().makeRequest(POST.toString(), "$ANALYZER_BASE_URI/refresh_search_analyzer")
+            fail("Expected a failure")
+        } catch (e: ResponseException) {
+            assertEquals("Unexpected RestStatus", RestStatus.BAD_REQUEST, e.response.restStatus())
+            val actualMessage = e.response.asMap()
+            val expectedErrorMessage = mapOf(
+                    "error" to mapOf(
+                            "root_cause" to listOf<Map<String, Any>>(
+                                    mapOf("type" to "parse_exception", "reason" to "request body is required")
+                            ),
+                            "type" to "parse_exception",
+                            "reason" to "request body is required"
+                    ),
+                    "status" to 400
+            )
```

Comment:
This seems to be not what you are trying to test? If testing for missing indices I'd expect to see an expected message for `throw IllegalArgumentException("Missing indices")`

Reply:
the uri change caused the exception behavior to change. I will revert back the uri and change this back